cmd/test.go

```diff
@@ -109,13 +109,12 @@ func opaTest(args []string) int {
 		Ignore: testParams.ignore,
 	}
 
-	var modules map[string]*ast.Module
-
-	var bundles map[string]*bundle.Bundle
-
-	var store storage.Store
-
-	var err error
+	var (
+		modules map[string]*ast.Module
+		bundles map[string]*bundle.Bundle
+		store   storage.Store
+		err     error
+	)
```

Comment:
I can be persuaded to revert this chunk, I just find the linter rule enforcing newlines between this horrendous 🙃

Reply:
Yeah, I don't really like it for vars either, and perhaps that's configurable. I do think it's pretty good for other things though, like separating return values, and so on.

Reply:
But this is fine with me!